site/production-checklist.xml

```diff
@@ -0,0 +1,291 @@
+
+      <doc:subsection name="resource-limits-disk-space">
+        <doc:heading>Free Disk Space</doc:heading>
+        <p>
+          Some free disk space should be available to avoid <a href="/disk-alarms.html">disk space alarms</a>.
+          By default RabbitMQ requires 50 MB of free disk space at all times. This improves developer
+          experience on many popular Linux distributions which may place the <code>/var</code> directory
+          on a small partition. However, this is not a value recommended for production environments,
+          since they may have significantly higher RAM limits.
+
+          Below are some basic guidelines for determining how much free disk space
+          is recommended:
+
+          <ul>
+            <li>At least 2 GB</li>
+            <li>
+              50% of the configured RAM limit when
+              the limit is between 1 and 8 GB of RAM
+            </li>
+            <li>
+              40% of the configured RAM limit when
+              the limit is between 8 and 32 GB of RAM
+            </li>
+            <li>
+              30% of the configured RAM limit when
+              the limit is above 32 GB of RAM
+            </li>
+          </ul>
+
+          The <code>rabbit.disk_free_limit</code> configuration setting
+          can be set to <code>{mem_relative, N}</code> to make it calculated
+          as a percentage of the RAM limit. For example, use <code>{mem_relative, 0.5}</code>
+          for 50%, <code>{mem_relative, 0.25}</code> for 25%, and so on.
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="resource-limits-file-handle-limit">
+        <doc:heading>Open File Handles Limit</doc:heading>
+        <p>
+          Operating systems limit maximum number of concurrently open file handles, which
+          incluldes network sockets. Make sure that your have limits set high enough to allow
+          for expected number of concurrent connections.
+
+          As a rule of thumb, multiple the 95th percentile number of concurrent connections
+          by 2 to calculate recommended open file handle limit.
+
+          See <a href="/networking.html">Networking guide</a>
+          for more information.
+        </p>
+      </doc:subsection>
+    </doc:section>
+
+
+    <doc:section name="security-considerations">
+      <doc:heading>Security Considerations</doc:heading>
+
+      <doc:subsection name="security-considerations-users-and-permissions">
+        <doc:heading>Users and Permissions</doc:heading>
+        <p>
+          See the section on vhosts, users, and credentials above.
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="security-considerations-erlang-cookie">
+        <doc:heading>Erlang Cookie</doc:heading>
+        <p>
+          It is highly recommended to restrict <a href="/clustering.html">Erlang cookie</a>
+          access only to the users that will run RabbitMQ and tools such as <code>rabbitmqctl</code>.
```

Comment:
It's not only recommended on Unix, it's mandatory: the Erlang VM refuses to start if the cookie is readable, writable or executable by other people than the owner, so a mode of 0400 is the minimum on Unix (it can be writable/executable by the owner).